DataFormats/Detectors/TRD/include/DataFormatsTRD/Tracklet64.h

```diff
@@ -148,6 +149,7 @@ class Tracklet64
  protected:
   uint64_t mtrackletWord; // the 64 bit word holding all the tracklet information for run3.
  private:
+  //  Tracklet& operator=(const Tracklet &rhs);   // not implemented
```

Comment:
This line might be a leftover from a rebase? If not the assignment operator should simply be `= default` I think

Reply:
made default